components/server/src/workspace/headless-log-service.ts

```diff
@@ -99,6 +99,7 @@ export class HeadlessLogService {
         const tasks = await new Promise<TaskStatus[]>((resolve, reject) => {
             const client = new StatusServiceClient(toSupervisorURL(wsi.ideUrl), {
                 transport: WebsocketTransport(),
+                debug: true,
```

Comment:
was a debug commit. Just wasted 40mins again to learn (again) we cannot test this PR on `core-dev` because of our nginx staging proxy... :see_no_evil: